### ISSUE TITLE
ModuleNotFoundError: No module named 'pve_exporter'

### ISSUE THREAD

**Issue by qcasey** [prometheus-pve/prometheus-pve-exporter#47]
Got this error when running the provided docker command. Is the dockerfile not importing the module correctly, or something a little more mysterious?

```
$ docker run -it --rm prompve/prometheus-pve-exporter --help

Traceback (most recent call last):
  File "/usr/bin/pve_exporter", line 5, in <module>
    from pve_exporter.cli import main
ModuleNotFoundError: No module named 'pve_exporter'
```

**Comment by znerol** [comment]
Thanks for the report. Looks like this broke around the 2.0.0 release. In the meantime, you can pull in `1.3.2`, that one seems to work:

```
$ docker run -it --rm prompve/prometheus-pve-exporter:1.3.2 --help
usage: pve_exporter [-h] [config] [port] [address]

positional arguments:
  config      Path to configuration file (pve.yml)
  port        Port on which the exporter is listening (9221)
  address     Address to which the exporter will bind

optional arguments:
  -h, --help  show this help message and exit
```

**Comment by znerol** [comment]
Uh, well. This is embarrassing. The `2.0.0` release on pypi.org is affected by the same bug as well.

**Comment by qcasey** [comment]
No, thank you! 1.3.2 works well

**Comment by znerol** [comment]
I released 2.0.1 which should fix the problem.

**Comment by qcasey** [comment]
Pulled latest which works great! Thanks a bunch
